### DISCUSSION TITLE
Calculating blood flow rate.

### DISCUSSION THREAD

**Discussion by Kokitake** [Open-Systems-Pharmacology/Forum#1806] [Questions & Problems]
![create_individual](https://github.com/user-attachments/assets/42f61bb0-bd0a-4ee5-bb11-d0a652cc86c0)
![blood flow rate](https://github.com/user-attachments/assets/55b481ef-2ea2-45e3-b856-765ad2dad42e)

Hello everyone.
I have a question about calculating blood flow rate.
When I select "Species; human" , "Gender; Male", "Age; 30 years", "Weight; 61.87 kg" , "Height=168.99 cm" and "BMI; 21.67" in "Create individual", the blood flow rate at Liver is shown as 0.39 l/hr.
In general, the hepatic blood flow rate in Japanese people is approximately 1.6 L/min. Why is this difference?
Could someone please give me an answer?

**Comment by PavelBal** [comment]
Blood flow to the liver is implemented as two separate routes - flow from the arterial blood, and the flow from the portal vein. The "Blood flow rate" parameter of the liver organ only reflects the arterial blood entering the liver. The total blood flow rate is the sum of this parameter and the blood flow rate of the portal vein (in your case, 1.1 l/min).

**Reply by Kokitake** [reply]
Thank you for your answer.
My question has been cleared up.
I'm a beginner in PK-sim, so I'll learn more.
